File: Day3/JavaScript/TaskOne.js
const prompt = require("prompt-sync")();

let number;

let secondNumber;

let operators;

let answer;


let count = 0

let counter = 0

function randomQuestions(){

number = Math.floor(Math.random()*100 + 1);

secondNumber = Math.floor(Math.random()*100 + 1);

operators = Math.floor(Math.random()*5 + 1);

}

console.log("======================================")



console.log("Welcome to Today's Arithmetic Quiz")

console.log("You have 30 minutes for this Test, GOODLUCK!!!")

console.log("===========================================")



function quiz(){

if(operators == 1){

userInput = parseInt(prompt(`What is ${number} + ${secondNumber}: `));

answer = number + secondNumber;

if(answer == userInput ){

console.log(" Correct ")

count++

}else{

console.log(" Incorrect ")

counter++

}


}

else if(operators == 2){

userInput = parseInt(prompt(`What is ${number} - ${secondNumber}: `));

answer = number - secondNumber;

if(answer == userInput ){

console.log(" Correct ")

count++

}else{

console.log(" Incorrect ")

counter++

}


}

else if(operators == 3){

userInput = parseInt(prompt(`What is ${number} * ${secondNumber}: `));

answer  = number * secondNumber;

if(answer == userInput ){

console.log(" Correct ")

count++

}else{

console.log(" Incorrect ")

counter++

}

}


else if(operators == 4){

userInput = parseInt(prompt(`What is ${number} % ${secondNumber}: `));

answer  = number % secondNumber;

if(answer == userInput ){

console.log(" Correct ")

count++

}else{

console.log(" Incorrect ")

counter++

}

}

else if(operators == 5){

userInput = parseInt(prompt(`What is ${number} / ${secondNumber}: `));

answer  = number / secondNumber;

if(answer == userInput ){

console.log(" Correct ")

count++

}else{

console.log(" Incorrect ")

counter++

}

}

}

for(let index = 1; index <= 10; index++){

randomQuestions()

quiz()

}

console.log(`You got ${count} correct questions and and failed      ${counter}  questions`)

if(count > 5){

console.log('Congratulations, You passed the test!!')

}else if(counter < 5){

console.log('You failed the test. Better luck next time')


}

File: Day3/Python/TaskOne.py
import random

print("===========================================")

print("Welcome to Today's Arithmetic Quiz")
print("You have 30 minutes for this Test, GOODLUCK!!!")

print("===========================================")


def quiz():

	count = 0

	counter = 0

	first_number = random.randint(1,100)

	second_number = random.randint(1,100)

	operator = random.randint(1,5)

	if operator == 1:

		userInput = int(input(f"What is {first_number} + {second_number}: "))

		answer = first_number + second_number

		if answer == userInput:

			print(" Correct ")

			count = count + 1

		else:

			print("Incorrect")

			counter = counter + 1


	elif operator == 2:

		userInput = int(input(f"What is {first_number} - {second_number}: "))

		answer = first_number + second_number;

		if answer == userInput:

			print(" Correct ")

			count = count + 1

		else:

			print("Incorrect")

			counter = counter + 1


	elif operator == 3:

		userInput = int(input(f"What is {first_number} * {second_number}: "))

		answer = first_number * second_number;

		if answer == userInput:

			print(" Correct ")

			count = count + 1

		else:
			print("Incorrect")

			counter = counter + 1

	elif operator == 4:

		userInput = int(input(f"What is {first_number} / {second_number}: "))

		answer = first_number / second_number;

		if answer == userInput:

			print(" Correct ")

			count = count + 1

		else:
			print("Incorrect")
			
			counter = counter + 1

	elif operator == 5:

		userInput = int(input(f"What is {first_number} % {second_number}: "))

		answer = first_number % second_number;

		if answer == userInput:

			print(" Correct ")

			count = count + 1

		else:
			print("Incorrect")
			
			counter = counter + 1


for index in range(10):

	quiz()


print(f"You got {count} correct questions and and failed      {counter} questions ")

if count > 5:

	print('Congratulations, You passed the test!!')

elif counter < 5:

	print('You failed the test. Better luck next time')







	

	




